docs/Reference/API-Objects.md

```diff
@@ -177,12 +177,15 @@ and
 |----------------------|:-------------------:|--------------------------------------------------------------------------------------------|
 | **blockHash**        | Data, 32&nbsp;bytes | Hash of the block containing this transaction. `null` when transaction is pending.         |
 | **blockNumber**      | Quantity            | Block number of the block containing this transaction. `null` when transaction is pending. |
+| **chainId**          | Quantity            | [Chain ID](../Concepts/NetworkID-And-ChainID.md).                                          |
```

Comment:
I just spotted typo a few lines up `eth_getTransactionsByBlockNumberAndIndex`

should be `eth_getTransactionByBlockNumberAndIndex` - I think it's related enough to this change to fix in the same PR